src/Sulu/Bundle/SecurityBundle/Entity/User.php

```diff
@@ -244,4 +244,54 @@ public function getFullName()
         return null !== $this->getContact() ?
             $this->getContact()->getFullName() : $this->getUsername();
     }
+
+    /**
+     * @VirtualProperty
+     * @Groups({"profile"})
+     *
+     * @return string
+     */
+    public function getFirstName()
+    {
+        return $this->contact->getFirstName();
+    }
+
+    /**
+     * @VirtualProperty
+     * @Groups({"profile"})
+     *
+     * @return string
+     */
+    public function getLastName()
+    {
+        return $this->contact->getLastName();
+    }
+
+    /**
+     * Set lastName.
+     *
+     * @param $lastName
+     *
+     * @return $this
+     */
+    public function setLastName($lastName)
+    {
+        $this->contact->setLastName($lastName);
+
+        return $this;
+    }
+
+    /**
+     * Set firstName.
+     *
+     * @param $firstName
+     *
+     * @return $this
+     */
+    public function setFirstName($firstName)
+    {
+        $this->contact->setFirstName($firstName);
+
+        return $this;
+    }
```

Comment:
Usually we put the getters and setters for the same property next to each other.